src/realm/table.cpp

```diff
@@ -3702,7 +3708,8 @@ void Table::change_nullability_list(ColKey key_from, ColKey key_to, bool throw_o
                     }
                     else {
                         if (throw_on_null) {
-                            throw realm::LogicError(realm::LogicError::column_not_nullable);
+                            throw std::runtime_error(util::format("Table '%1' column '%2' has null value(s) in list",
+                                                                  get_name(), get_column_name(key_from)));
```

Comment:
how about printing the list's column name as well?

Reply:
I am doing that - I think

Reply:
yes you are, my mistake 🙄